keyboards/avalanche/v4/v4.c

```diff
@@ -0,0 +1,4 @@
+// Copyright 2022 Vitaly Volkov (@vlkv)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "avalanche.h"
```

Comment:
This allows you to have a default implementation, without needing to specify a config, and allows for the user code to override the behavior.

This is especially useful when compiling from the QMK Configurator. 

```suggestion
#include "avalanche.h"

#ifdef OLED_ENABLE
oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
    return OLED_ROTATION_180;
}

bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
    oled_write_P(PSTR("Avalanche\nVersion 4"), false);
    return true;
}
#endif
```